### DISCUSSION TITLE
Geb is no longer maintained

### DISCUSSION THREAD

**Discussion by vntouzepis** [spockframework/spock#1993] [Q&A] [answered]
Hello,

It seems that Geb is no longer maintained https://www.gebish.org/
Does anyone know if there is a possibility Geb will continue to be maintained, or we have to turn in other solutions for UI tests with Spock, i.e. Selenium?

Thank you in advance

**Comment by Vampire** [comment, marked as answer]
I'm not sure why you ask here.
Spock is Spock, Geb is Geb.
It is a totally different project that just happens to have a nice integration with Spock.

Geb is open source, so like with any other open source project, if the original maintainer abandons the project, you always have the option to overtake the project if he permits, or to create a fork and maintain that. Maybe he also just needs co-maintainers to keep it on. 🤷‍♂️

**Reply by vntouzepis** [reply]
Thank you very much for your response!
Indeed Spock is independable from Spock, but I couldn't find a more related discussion place.
It seems a bit strange to me that Geb is no longer maintained, as it is excellent integration with Spock, and it is used from many companies with big codebase.

**Reply by Vampire** [reply]
Well, make those companies give the maintainer money, maybe he will continue maintenance then.
Or make those companies donate work-time of employees to take on maintenance tasks.
I'm also sad to see it gone.

A more appropriate place is probably something that is anyhow related to that project, more than just being integrated with it. :-D
For example its dev mailing list at https://groups.google.com/g/geb-dev.
Or - as discussions are not enabled - its issues at https://github.com/geb/issues/issues

It also seems there is at least one person stepping up to take over maintenance or a fork: https://github.com/geb/issues/issues/692

**Comment by leonard84** [comment]
Geb has been adopted by Apache https://github.com/apache/groovy-geb
